sr_port/gtmrecv_comm_init.c

```diff
@@ -98,6 +99,7 @@ int gtmrecv_comm_init(in_port_t port)
 	{
 		rts_error_csa(CSA_ARG(NULL) VARLSTCNT(7) ERR_REPLCOMM, 0, ERR_TEXT, 2,
 				 RTS_ERROR_LITERAL("Could not bind local address"), ERRNO);
+		freeaddrinfo(ai_ptr);
```

Comment:
Good catch. The rts_error_csa() will not return since REPLCOMM is of severity "error" (not "info"). So the freeaddrinfo() needs to be done before the rts_error(). Also I notice a couple of rts_error_csa() calls before this which need to (but do not) do the freeaddrinfo() as well. Will fix all of those.